joint1 :
   return_delay_time     : 1
   operating_mode        : 3

joint2 :
   return_delay_time     : 1
   operating_mode        : 3

joint3 :
   return_delay_time     : 1
   operating_mode        : 3

joint4 :
   return_delay_time     : 1
   operating_mode        : 3

joint5 :
   return_delay_time     : 1
   operating_mode        : 3

joint6 :
   return_delay_time     : 1
   operating_mode        : 3
